pkg/inst/tests/test_rdd.R

```diff
@@ -314,6 +314,20 @@ test_that("values() on RDDs", {
   expect_equal(actual, lapply(intPairs, function(x) { x[[2]] }))
 })
 
+test_that("pipeRDD() on RDDs", {
+  if (.Platform$OS.type == "unix") {
+    actual <- collect(pipeRDD(rdd, "cat"))
+    expected <- as.list(as.character(1:10))
+    expect_equal(actual, expected)
+  }
```

Comment:
either is fine by me - if `more`works on Windows as well, then lets go with `more`